README.md

```diff
@@ -1,118 +1,128 @@
 <div align="center">
-  <img src="./logo/logo-web.svg" width="348.61" height="100" alt="SVGO logo"/>
+  <img src="./logo/logo-web.svg" width="348.61" height="100" alt=""/>
 </div>
 
-## SVGO [![npm version](https://img.shields.io/npm/v/svgo)](https://npmjs.org/package/svgo) [![Discord](https://img.shields.io/discord/815166721315831868)](https://discord.gg/z8jX8NYxrE)
+# SVGO [![npm](https://img.shields.io/npm/v/svgo)](https://npmjs.org/package/svgo) [![chat](https://img.shields.io/discord/815166721315831868)](https://discord.gg/z8jX8NYxrE) [![docs](https://img.shields.io/badge/docs-svgo.dev-blue)](https://svgo.dev/)
 
-**SVG O**ptimizer is a Node.js-based tool for optimizing SVG vector graphics files.
+SVGO, short for **SVG O**ptimizer, is a Node.js library and command-line application for optimizing SVG files.
 
 ## Why?
 
-SVG files, especially those exported from various editors, usually contain a lot of redundant and useless information. This can include editor metadata, comments, hidden elements, default or non-optimal values and other stuff that can be safely removed or converted without affecting the SVG rendering result.
+SVG files, especially those exported from vector editors, usually contain a lot of redundant information. This includes editor metadata, comments, hidden elements, default or suboptimal values, and other stuff that can be safely removed or converted without impacting rendering.
 
 ## Installation
 
-Via npm:
-```sh
-npm -g install svgo
-```
-Via yarn:
+You can install SVGO globablly through npm, yarn, or pnpm. Alternatively, drop the global flag (`global`/`-g`) to use it in your Node.js project.
+
 ```sh
+# npm
+npm install -g svgo
```

Comment:
These days npx, yarn dlx or pnpx are more popular ways to run something globally.

Reply:
Sorry for the quick merge! I should'be left this open clearly. ^-^'

I actually opted for the individual commands intentionally because while looking `npx`, I've found it's very slow. This checks out with the fact that typically when I see it, it's for running one-time tasks like `create-react-app`. 

Every time one does `npx svgo`, it will check through node modules etc, taking 2 seconds just to run `npx svgo --help`. Not the ideal UX, so I thought it'd be best to document them separately, which is also handy to explain how to depend on it in a Node.js project anyway.